kiwi/system/kernel.py

```diff
@@ -66,6 +65,12 @@ def get_kernel(self, raise_on_not_found=False):
                     kernel_file_to_check = kernel_file_to_check.replace(
                         'zImage', 'vmlinux'
                     ) + '.gz'
+                if 'vmlinuz' in kernel_file_to_check:
+                    new_file_to_check = kernel_file_to_check.replace(
```

Comment:
The tests missing for those lines 69~73 in `test/unit/system_kernel_test.py`

Reply:
I'll try my best ...

Reply:
looks good now ?

Reply:
Yes! Thanks much :+1: